frontend/templates/base.html

```diff
@@ -1,13 +1,18 @@
 <!DOCTYPE html>
+{% load static tailwind_tags %}
+
 <head>
     <meta charset="UTF-8">
-    {% block title %} Foo bar baz{% endblock %}
+    {% tailwind_css %}
+    {% block title %}Foo bar baz{% endblock %}
```

Comment:
Looks like this is supposed to be wrapped in the title tag
```suggestion
    <title>{% block title %}Foo bar baz{% endblock %}</title>
```